eng/build.ps1

```diff
@@ -85,6 +85,7 @@ if ($vs) {
 
     # This tells .NET Core to use the same dotnet.exe that build scripts use
     $env:DOTNET_ROOT="$PSScriptRoot\..\artifacts\bin\testhost\netcoreapp5.0-Windows_NT-$configuration-$archTestHost";
+    $env:DEVPATH="$PSScriptRoot\..\artifacts\bin\testhost\net472-Windows_NT-$configuration-$archTestHost";
```

Comment:
Is DEVPATH just a thing of Full Framework? If not should we condition it out for netcoreapp and other tfms?

Reply:
Yeah that's .NET Framework only.